### PR TITLE
Pre-merge Cleanup

### PR DESCRIPTION
**Pull request by martinmaly**
* Tidy go modules
* Fix year in license header

### CODE REVIEW

**Comment by martinmaly** [comment]
@droot If your preference is to merge this cleanup into porch branch _before_ merging porch into main, let's merge this PR first and I'll re-do the merge porch --> main PR.
Otherwise I can also merge the big PR first and then do this after. LMK

File: site/sidebar.md [hunk 1]
```diff
@@ -64,10 +64,6 @@
     - [ResourceList](reference/schema/resource-list/)
     - [CRD Status Convention](reference/schema/crd-status-convention/)
     - [Config Connector Status Convention](reference/schema/config-connector-status-convention/)
-  - [Annotations Reference](reference/annotations/)
```
Comment (droot) [inline site/sidebar.md:67]: This shouldn't be removed. This is a generated file. Looks like `porch` branch doesn't have the sidebar template updated.

**Comment by droot** [comment]
> @droot If your preference is to merge this cleanup into porch branch _before_ merging porch into main, let's merge this PR first and I'll re-do the merge porch --> main PR. Otherwise I can also merge the big PR first and then do this after. LMK

Since it's a minor cleanup, I would prefer doing it after our big PR merge. Will be reviewing the big PR soon.

File: site/sidebar.md (hunk 1 above)
Comment (droot) [reply site/sidebar.md:67]: Oh, I see it now. In that case, may be merging this PR first sounds reasonable.

**Review by droot — APPROVED** [review]

File: site/sidebar.md (hunk 1 above)
Comment (martinmaly) [reply site/sidebar.md:67]: Thanks

**Pull request merged**